frontend/src/views/c-zoom.vue

```diff
@@ -132,6 +132,16 @@
       .body(v-if="slice.messageBody !== ''", v-show="slice.isOpen")
         pre {{ slice.messageBody }}
           .dashed-border
+      template(
+        v-if="showDiffstat"
+        v-for="(widths, color) in getContributionBars(slice)"
+      )
+        c-stacked-bar-chart(
+          v-bind:bars="widths.map((width) => {\
+            return {width: width,\
+                    color: color}\
+          })"
+        )
```

Comment:
Since the `c-stacked-bar-chart` component is now accepting an array of bars, I don't think it makes sense to render two different stacked-bar-chart components (one for green, one for red) - or in the case of the `checkedFileTypeContributionBars` in `c-summary-charts`, rendering different stacked-bar-charts for each file type.

Can the `getContributionBars` and `getFileTypeContributionBars` be refactored to return all the bars needed for the chart instead?

 i.e. instead of `getContributionBars` returning `{ red: [40], limegreen: [60, 80] }` and constructing a different `stacked-bar-chart` for red and limegreen, can it instead return 
```typescript
[
    { color: "red", width: 40 },
    { color: "limegreen", width: 60 },
    { color: "limegreen", width: 80 },
]